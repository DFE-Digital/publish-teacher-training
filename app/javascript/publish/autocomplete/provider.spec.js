/**
 * @jest-environment jsdom
 */

import init from './provider'

describe('Provider', () => {
  describe('init', () => {
    beforeEach(() => {
      document.body.innerHTML = `
         <div id="outer-container">
           <input type="text" id="provider">
           <div id="provider-autocomplete"></div>
         </div>
       `

      init()
    })

    it('should instantiate an autocomplete', () => {
      expect(document.querySelector('#outer-container')).toMatchSnapshot()
    })
  })
})
