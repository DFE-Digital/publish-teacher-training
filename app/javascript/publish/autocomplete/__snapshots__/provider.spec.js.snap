// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Provider init should instantiate an autocomplete 1`] = `
<div
  id="outer-container"
>
  
           
  <input
    id="provider"
    type="text"
  />
  
           
  <div
    id="provider-autocomplete"
  />
  
         
</div>
`;
